php/validate.php

```diff
@@ -0,0 +1,16 @@
+<?php
+ $username = $_POST['username'];
+ $password = $_POST['password'];
+ 
+ if ($username === "asteroide77" && $password === "secret") {
```

Comment:
It would be better if we read the passwords from the environment to prevent secrets leaking!